apps/sms/views/shared/style/composer.css

```diff
@@ -34,6 +34,7 @@ form[role="search"] button[type="submit"] + p > textarea {
   -moz-border-end: 0.1rem solid #ccc;
 
   font-size: 1.7rem;
+  font-family: "Firefox Emoji", sans-serif;
```

Comment:
You need to add this font into few more places @pivanov :
* for the subject field displayed in the Conversation view;
* for the subject field displayed in Report view

I'm playing with the patch on my device currently :)